### ISSUE TITLE
Find in Files failing

### ISSUE THREAD

**Issue by Naatan** [Komodo/KomodoEdit#57]
Reference: https://bugs.activestate.com/show_bug.cgi?id=106442

```
[2015-03-25 14:15:41,467] [ERROR] console-logger: TypeError: widget is null (2)
in chrome://komodo/content/find/find2.js:1428
Traceback (most recent call last):
  File "chrome://komodo/content/find/find2.js", line 1428, in 

[2015-03-25 14:16:03,298] [ERROR] find.dialog: 
-- EXCEPTION START --
TypeError: widget is null
+ stack
    _collapse_widget@chrome://komodo/content/find/find2.js:1428:9
    update@chrome://komodo/content/find/find2.js:268:1
    _init@chrome://komodo/content/find/find2.js:1149:5
    on_load@chrome://komodo/content/find/find2.js:89:9
    onload@chrome://komodo/content/find/find2.xul:1:1
-- EXCEPTION END --
```

**Comment by Naatan** [comment]
I'm unable to reproduce, can anyone provide steps to reproduce?

**Comment by cgchoffman** [comment]
I'm fairly certain this is resolved by #79.  [Fix references](https://github.com/Komodo/KomodoEdit/commit/32ed693246ad9958dc4b3629d5e305ba0979c6f5#diff-fe8988e698b4c5cd70b71f21415aba64L1249)
